### PR TITLE
[ResourceBundle] Api chapter

### PR DESCRIPTION
**Pull request by aRn0D**
Q  | A
----|----
Doc fix? | no
New docs? | yes
Fixed tickets |  -

### CODE REVIEW

**Comment by aRn0D** [comment]
ping @pjedrzejewski @stloyd

File: bundles/SyliusResourceBundle/api.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+The previous header will be translated to :
+
+.. code-block:: php
+
+    $handler = $this->get('fos_rest.view_handler');
+    $handler->setExclusionStrategyVersion('1.0.1');
+    $handler->setExclusionStrategyGroups(array('Sylius', 'Details'));
+
+In serializer configuration files, you can filter the data that you want to send by using the field called `since_version`, `until_version` and `group`.
+The following example show you how it works. The lastname won't be return after the version 0.3 and the firstname will be returned after the version 0.2.
```
Comment (stloyd) [inline bundles/SyliusResourceBundle/api.rst]: Variables & versions should also be somehow marked as _other_ content.

File: bundles/SyliusResourceBundle/api.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                type: string
+                since_version: 0.2
+                groups: [Sylius]
+            lastName:
+                expose: true
+                type: string
+                until_version: 0.3
+                groups: [Details]
+
+You can find extra information `there <http://jmsyst.com/libs/serializer>`_.
```
Comment (stloyd) [inline bundles/SyliusResourceBundle/api.rst]: > More details about serializer can be found in its official `documentation <http://jmsyst.com/libs/serializer>`_.

File: bundles/SyliusResourceBundle/api.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+Api Loader
+----------
+
+The Api Loader generate your API routes automatically. The ``resource`` is the concatenation of the application and resource name.
```
Comment (stloyd) [inline bundles/SyliusResourceBundle/api.rst]: > Routes for your API will be automatically generated by `ApiLoader` class.

**Pull request merged**